### DISCUSSION TITLE
Please include a automated tutorial

### DISCUSSION THREAD

**Discussion by SNUBNOZEMUSIC** [community/community#65729] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I really need help without having to pay someone, if it's possible, to please update a version where we can watch a step by step comprehensive easy to watch and learn tutorials. To save time and energy. I have had it up to here as I lost out on a free membership to actually launch my first business and gain capital. Also there is MYSTERIOUSLY A (FROM SKRATCH) Pretending to be my company (I almost chose it to sign into my AWS accounts, please help us catch these crooks it's a team of them (I suspect londontv UK guy) 
as he reached out early on during COVID and since my information has been leaked it's a cold operation going on and been going on behind my back PLEASE WDEACTIVATE OR TELL ME HOW I INVESTIGATE THAT PARTICULAR NAME AND SERVER

**Comment by Binary-Ninja-007** [comment]
Good.
